app/controllers/discourse_ai/ai_helper/assistant_controller.rb

```diff
@@ -161,7 +161,11 @@ def get_post_param!
       end
 
       def rate_limiter_performed!
-        RateLimiter.new(current_user, "ai_assistant", 6, 3.minutes).performed!
+        if action_name == "caption_image"
+          RateLimiter.new(current_user, "ai_assistant_caption_image", 20, 1.minutes).performed!
```

Comment:
```suggestion
          RateLimiter.new(current_user, "ai_assistant_caption_image", 20, 1.minute).performed!
```

Tiny detail. (Both work, but `1.minute` reads better.)